README.release.md

```diff
@@ -0,0 +1,49 @@
+# Creating a New Release
+
+## Changelog
+
+The changelog is autogenerated by github at the release notes by inspecting 
+labels at PRs.
+
+To have a proper changelog, make sure the PRs are labeled as expected so they 
+they appear at proper the proper section according to [1] this are the labels:
```

Comment:
- `they they appear at proprt the proper section` is broken.
- You also need a `.` before stating with "according...".
- this => these